vespa-athenz/src/main/java/com/yahoo/vespa/athenz/api/AthenzRole.java

```diff
@@ -20,6 +22,16 @@ public AthenzRole(String domain, String roleName) {
         this.roleName = roleName;
     }
 
+    public static AthenzRole fromString(String string) {
```

Comment:
consider naming the method to reflect that it expects the format <domain>:role.<rolename>

Reply:
Addressed in new commit.